.github/workflows/handle-release-branches.yml

```diff
@@ -0,0 +1,126 @@
+name: Handle Release Branches
+
+on:
+  push:
+
+jobs:
+  branch-checks:
+    runs-on: ubuntu-latest
+    steps:
+      - id: get-branch
+        run: |
+          BRANCH=($(echo ${{ github.ref }} | sed -E 's/refs\/heads\///'))
+          echo "branch=$BRANCH" >> $GITHUB_ENV
+    outputs:
+      branch: ${{ env.branch }}
+      is-latest-branch: ${{ env.branch == 'main' }}
+      is-next-branch: ${{ env.branch == 'next' }}
+      is-release-branch: ${{ startsWith(env.branch, 'release-') }}
+      is-actionable-branch: ${{ env.branch == 'main' || env.branch == 'next' || startsWith(env.branch, 'release-') }}
+  
+  get-latest-release-branch:
+    needs: branch-checks
+    if: ${{ needs.branch-checks.outputs.is-latest-branch == 'true' || needs.branch-checks.outputs.is-release-branch == 'true' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: main
+          
+      - id: latest-version
+        uses: notiz-dev/github-action-json-property@release
+        with:
+          path: ${{ github.workspace }}/main/docs/versions/latest.json
+          prop_path: version
+          
+      - run: |
+          LATEST_RELEASE_BRANCH=($(echo ${{ steps.latest-version.outputs.prop }} | sed -E 's/([0-9]+)\.([0-9]+).*/release-\1-\2/'))
+          echo "latest-release-branch=$LATEST_RELEASE_BRANCH" >> $GITHUB_ENV
+    outputs:
+      branch: ${{ env.latest-release-branch }}
+  
+  create-latest-release-branch:
```

Comment:
Rather than creating a branch for this case, it would be far more straightforward^1 to just use the Netlify webhook directly (as is currently done in the `build-frontpage` script).

[1] The latest version of the docs will be the non-versioned production build of the site, so trying to build that from a branch other than `master` would be more complex than necessary.